README.md

```diff
@@ -23,7 +23,109 @@ At present the project encompasses these components
 - [MICM](https://github.com/NCAR/micm)
     - Model Independent Chemical Module
 
-## Available grids
+# Installation
+
+## Pip
+```
+pip install musica
+```
+## CMake
```

Comment:
I think we can use either `pip` or `cmake` to install MUSICA, but not both at the same time? If so, better to add sentence to clarify that.